drivers/SmartThings/deepsmart/src/lifecycles.lua

```diff
@@ -16,11 +16,23 @@ function lifecycle_handler.init(driver, device)
   if (is_bridge) then
     -- just add wiser(bridge) to wisers
     wisers.add_wiser(device.device_network_id, device:get_field(config.FIELD.IP), device)
+    -- if exist bridge then start ssdp ip check schedule
+    if (wisers.ip_check_state == nil) then
+      wisers.ip_check_state = 1
+      -- start ssdp schedule to check bridge ip
+      driver:call_on_schedule(
+      600,
+      function ()
+        return wisers.check_ip(discovery)
+      end,
+      'Check ip schedule')
+    end
```

Comment:
This can exist outside the handler; you dont need to prevent it from running when there isn't any devices, because the driver should be running only when there are device, or when it is actively trying to discover the first device (which will complete before the 10 minutes is up).

The benefit of that is you dont need `wisers.ip_check_state` field to prevent it from being setup twice for the first device, and it clearer that only one task is being run in the driver (rather than a task per device).

Reply:
move it to init.lua and delete ip_check_state

Reply:
This should be removed from the init lifecycle handler as well.

Reply:
remove it